### PR TITLE
feat: initial application setup for dremio

### PR DESCRIPTION
**Pull request by jonathonbyrdziak**
## Summary

This is a bare bones initial setup for Dremio. Still need to ensure that everything is configured for a basic Dremio deployment.

## Checklist

- [ ]  No images hosted from dockerhub
- [ ]  Are dashboards present to understand the health of the application.  There **must** be at least 1 of these
    - [x]  all databases should have dashboards
    - [x]  ideally also have at least cpu/mem utilization dashboards for webserver tier of the app
    - [ ]  you can use `plural from-grafana` to convert a grafana dashboard found via google to our CRD
- [ ]  Are scaling runbooks present
    - [ ]  all databases **must** have scaling runbooks
    - [ ]  you can use the charts in `pluralsh/module-library` to accelerate this
- [ ]  do you need to add config overlays?
    - [ ]  inputing secrets
    - [ ]  configuring autoscaling
- [ ]  If there’s a web-facing component to the app, we need to support OIDC authentication and setting up private networks if no authentication option is viable
- [x]  All major clouds must be supported
    - [x]  Azure
    - [x]  AWS
    - [x]  GCP

### CODE REVIEW

File: dremio/Pluralfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+REPO dremio
+ATTRIBUTES CapitalAudience repository.yaml
```
Comment (michaeljguarino) [inline dremio/Pluralfile:2]: This should be `ATTRIBUTES Plural repository.yaml` (unfortunately plural can be the only publisher in this repo)

Comment (jonathonbyrdziak) [reply dremio/Pluralfile]: Okay that was fixed @michaeljguarino

**Review by michaeljguarino — APPROVED** [review]

File: dremio/repository.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+name: dremio
+description: dremio deployed on plural
+category: DATA
+private: true
+icon: plural/icons/dremio.png
```
Comment (michaeljguarino) [inline dremio/repository.yaml:5]: actually just realized, an icon is needed here for the catalog listing

Comment (jonathonbyrdziak) [reply dremio/repository.yaml:5]: Added the dremio icon @michaeljguarino

**Review by michaeljguarino — APPROVED** [review]

**Pull request merged**